backend/api/routes.py

```diff
@@ -266,25 +282,104 @@ def get_autocompletion(text: str):
     return autocomplete(text, repo=database)
 
 
-@app.get("/database/tables", tags=["database"])
-def get_table_names():
+@app.get("/terminologies", tags=["embeddings"])
+def get_terminologies():
     """
-    Get all table names available in the database.
```

Comment:
If you put docstrings into the REST annotation instead (via description) it is also visible to the API user

Reply:
It should be visible to the user with this notation too, but I can change it.

Reply:
Thats new to me, but if it works we can keep it like it is